analytics/src/main/kotlin/uk/govuk/app/analytics/AnalyticsClient.kt

```diff
@@ -1,59 +1,74 @@
 package uk.govuk.app.analytics
 
+import com.google.firebase.analytics.FirebaseAnalytics
 import uk.gov.logging.api.analytics.AnalyticsEvent
 import uk.gov.logging.api.analytics.logging.AnalyticsLogger
-import uk.gov.logging.api.analytics.parameters.ButtonParameters
-import uk.gov.logging.api.analytics.parameters.ScreenViewParameters
-import uk.govuk.app.analytics.search.SearchParameters
+import java.util.Locale
 import javax.inject.Inject
 
 class AnalyticsClient @Inject constructor(
     private val analyticsLogger: AnalyticsLogger
 ): Analytics {
 
-    companion object {
-        private const val WIDGET_ACTION = "Widget"
-    }
-
-    override fun screenView(screenClass: String, alias: String, title: String) {
+    override fun screenView(screenClass: String, screenName: String, title: String) {
         analyticsLogger.logEvent(
             true,
-            AnalyticsEvent.screenView(
-                ScreenViewParameters(
-                    clazz = screenClass,
-                    name = alias,
-                    title = title
+            AnalyticsEvent(
+                eventType = FirebaseAnalytics.Event.SCREEN_VIEW,
+                parameters = mapOf(
+                    FirebaseAnalytics.Param.SCREEN_CLASS to screenClass,
+                    FirebaseAnalytics.Param.SCREEN_NAME to screenName,
+                    "screen_title" to title,
+                    "language" to Locale.getDefault().language
```

Comment:
Definitely not a blocker, but maybe we should refactor this into a function at some point 🤔